packages/http/src/validator/validators/utils.ts

```diff
@@ -2,7 +2,7 @@ import { IPrismDiagnostic } from '@stoplight/prism-core';
 import { DiagnosticSeverity, Segment } from '@stoplight/types';
 import { getSemigroup } from 'fp-ts/lib/NonEmptyArray';
 import { getValidation } from 'fp-ts/lib/Either';
-import { option } from 'fp-ts/lib/Option';
+import { option, tryCatch, Option } from 'fp-ts/lib/Option';
```

Comment:
I would try to follow our code-style here and
```suggestion
import * as Option from 'fp-ts/lib/Option';
```

Especially when there are functions from different buckets.